pilot/pkg/model/push_context.go

```diff
@@ -1710,6 +1727,14 @@ func (ps *PushContext) initTelemetry(env *Environment) (err error) {
 	return
 }
 
+func (ps *PushContext) initProxyConfigs(env *Environment) (err error) {
+	if ps.ProxyConfigs, err = GetProxyConfigs(env.IstioConfigStore, env.Mesh()); err != nil {
+		pclog.Errorf("failed to initialize proxy configs: %v", err)
+		return
```

Comment:
nit: I would not use named return types, it just makes things a bit more confusing.

Reply:
fixed